cli/pkg/kctrl/cmd/package/installed/create_or_update.go

```diff
@@ -40,7 +42,8 @@ type CreateOrUpdateOptions struct {
 	depsFactory cmdcore.DepsFactory
 	logger      logger.Logger
 
-	WaitFlags cmdcore.WaitFlags
+	WaitFlags       cmdcore.WaitFlags
+	YttOverlayFlags YttOverlayFlags
```

Comment:
Maybe group it with valuesFile instead?

Reply:
Grouped it with WaitFlags as it is also a set of flags used across multiple commands
Happy to move it tho, no strong opinions